lfs/lfs.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/rubyist/tracerx"
 )
 
-const (
-	Version = "1.5.0"
```

Comment:
Turns out that when I import `github.com/git-lfs/git-lfs/lfs` in `config/version.go` in order to use `lfs.Version` I get `import cycle not allowed` :-( Any suggestion how to solve that?